### PR TITLE
stat/card: fix rho functions

### PR DESCRIPTION
**Pull request by kortschak**
Of course there was an error.

Please take a look.

### CODE REVIEW

File: stat/card/card_test.go [hunk 1]
```diff
@@ -135,7 +135,7 @@ func TestUnion(t *testing.T) {
 			t.Errorf("unexpected error from Union call: %v", err)
 		}
 		if got := u.Count(); !floats.EqualWithinRel(got, 2*test.count, 2*test.tol) {
```
Comment (sbinet) [inline stat/card/card_test.go]:
```suggestion
		if got, want := u.Count(), 2*test.count; !floats.EqualWithinRel(got, want, 2*test.tol) {
```

File: stat/card/card_test.go [hunk 2]
```diff
@@ -135,7 +135,7 @@ func TestUnion(t *testing.T) {
...
-			t.Errorf("unexpected count for %s: got:%.0f want:%.0f", test.name, got, test.count)
+			t.Errorf("unexpected count for %s: got:%.0f want:%.0f", test.name, got, 2*test.count)
```
Comment (sbinet) [inline stat/card/card_test.go]:
```suggestion
			t.Errorf("unexpected count for %s: got:%.0f want:%.0f", test.name, got, want)
```

File: stat/card/card_test.go (hunk 1 above)
Comment (kortschak) [reply stat/card/card_test.go]: I don't think these are necessary.

**Comment by kortschak** [comment]
It pops out when the more complex estimations are used.

**Pull request merged**